[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-1 (#114270)
openshift/openshift-docs: modules/oc-compliance-rerunning-scans.adoc

[id="re-running-scans_{context}"]
= Re-running scans

[role="_abstract"]

> Although it is possible to run scans as scheduled jobs, you must often re-run a scan on demand, particularly after remediations are applied or when other changes to the cluster are made.

Reviewer: ```suggestion
Although it is possible to run scans as scheduled jobs, you must often re-run a scan on demand, particularly after remediations are applied or when other changes to the cluster are made.

Reruning a scan with the Compliance Operator requires the use of an annotation on the scan object. However, with the `oc-compliance` plugin you can rerun a scan with a single command.
```

.Procedure

> . Rerun the scans for a `ScanSettingBinding` object:

Reviewer 2: Single step procs get a `*`

> * Rerun a scan with the Compliance Operator that requires use of an annotation on the scan object. However, with the `oc-compliance` plugin you can rerun a scan with a single command. Enter the following command to rerun the scans for the `ScanSettingBinding` object named `my-binding`:

Reviewer: ```suggestion
*  Rerun the scans for the `ScanSettingBinding` object named `my-binding` by running the following command:
```
